packages/imtr-client/src/models/checker.ts

```diff
@@ -78,6 +78,11 @@ export default class Checker {
       throw Error("Answers must be of type object");
     }
 
+    // Skip when answers are empty (to prevent error with checkers without questions to render)
```

Comment:
Are checkers without questions even supported? :-/

Reply:
Yes, checkout this one: http://localhost:3000/2eYY7McaiRgfnERo9/vragen-en-uitkomst

Reply:
So that means address leaves to rendering no questions because they are answered by autofill...